src/components/08-navigation/nav-primary.njk

```diff
@@ -9,11 +9,25 @@
       <span>{{ link.text }}</span>
     </button>
     <ul id="{{ nav.id_prefix }}{{ link.id }}" class="usa-nav-submenu{% if nav.mega %} usa-megamenu usa-grid-full{% endif %}">
-      {% for child in link.links %}
-      <li>
-        <a href="{{ child.href }}">{{ child.text }}</a>
-      </li>
-      {% endfor %}
+      {% if nav.id_prefix == "basic-" or nav.id_prefix == "extended-" %}
```

Comment:
could you do the `-%}` close style on these if/else statements in the njk files so the generated html samples in Fractal don't have a bunch of extra blank lines. 

There's an example in the button PR of how it works here https://github.com/18F/web-design-standards/pull/2098/files#diff-c95f7975ef5ea8e3cfc3fb942baa9055R2